onnxruntime/contrib_ops/cpu/murmur_hash3.cc

```diff
@@ -193,14 +197,14 @@ Status MurmurHash3::Compute(OpKernelContext* ctx) const {
       ++output;
     }
   } else {
-    auto input = reinterpret_cast<const uint32_t*>(keys->DataRaw());
-    const auto input_end = input + input_count;
+    auto input = reinterpret_cast<const unsigned char*>(keys->DataRaw());
```

Comment:
Why `unsigned char`? I remember Murmur hashing is based on `uint32`.

Reply:
cast to uint32 when calling MurmurHash3_x86_32